index.bs

```diff
@@ -6146,6 +6153,125 @@ The [=remote end steps=] with |command parameters| are:
 
 </div>
 
+#### The emulation.setUserAgentOverride Command ####  {#command-emulation-setUserAgentOverride}
+
+The <dfn export for=commands>emulation.setUserAgentOverride</dfn> command modifies
+User-Agent on the given top-level traversables or user contexts.
+
+<dl>
+   <dt>Command Type</dt>
+   <dd>
+    <pre class="cddl" data-cddl-module="remote-cddl">
+      emulation.SetUserAgentOverride = (
+        method: "emulation.setUserAgentOverride",
+        params: emulation.SetUserAgentOverrideParameters
+      )
+
+      emulation.SetUserAgentOverrideParameters = {
+        userAgent: text / null,
+        ? contexts: [+browsingContext.BrowsingContext],
+        ? userContexts: [+browser.UserContext],
+      }
+    </pre>
+   </dd>
+   <dt>Result Type</dt>
+   <dd>
+    <code>
+     EmptyResult
+    </code>
+   </dd>
+</dl>
+
+<div algorithm="updated `User-Agent` getter steps">
+The [=default User-Agent value=] getter steps is implementation defined. A
+WebDriver-BiDi [=remote end=] must have an implementation that runs the following
+steps:
+
+Issue: get |related navigables|.
```

Comment:
I found 2 usages of the ["default `User-Agent` value"](https://fetch.spec.whatwg.org/#default-user-agent-value):
1. [Set a header](https://fetch.spec.whatwg.org/#ref-for-default-user-agent-value) for network request.
2. [NavigatorId](https://html.spec.whatwg.org/#navigatorid)'s ["appVersion"](https://html.spec.whatwg.org/#dom-navigator-appversion) and ["userAgent"](https://html.spec.whatwg.org/#dom-navigator-useragent).

We need to fins a way to specify what "related navigables" means in those cases.

Reply:
I guess we could propose an update to the fetch and html specs? I think there is a way how find related navigable for an HTTP request (like in interception) and for a document, does it work here?

Reply:
Yeah, I think we should attempt creating patches for the HTML and Fetch spec. WDYT?

Reply:
For reference the algorithm we use to map request to navigable is https://www.w3.org/TR/webdriver-bidi/#get-navigable-for-request

To hook from the fetch spec, you can look at the fetch PR at https://whatpr.org/fetch/1540.html#concept-http-fetch (we still haven't landed this so it makes a bit hard to follow the BiDi/fetch integration)